### ISSUE TITLE
spago install with empty dependencies list does not get updated on install

### ISSUE THREAD

**Issue by justinwoo** [spacchetti/spago#281]
```
# deps is empty
$ cat spago.dhall
{-
Welcome to a Spago project!
You can edit this file as you like.
-}
{ name =
    "pscid"
, dependencies =
    [] : List Text
, packages =
    ./packages.dhall
, sources =
    [ "src/**/*.purs" ]
}

# installing a thingy
$ spago install transformers
Installation complete.

# deps is still empty
$ cat spago.dhall
{-
Welcome to a Spago project!
You can edit this file as you like.
-}
{ name =
    "pscid"
, dependencies =
    [] : List Text
, packages =
    ./packages.dhall
, sources =
    [ "src/**/*.purs" ]
}
```

deps still empty after installationl

**Comment by justinwoo** [comment]
```
$ cat spago.dhall
{-
Welcome to a Spago project!
You can edit this file as you like.
-}
{ name =
    "pscid"
, dependencies =
    [ "prelude" ]
, packages =
    ./packages.dhall
, sources =
    [ "src/**/*.purs" ]
}
$ spago install console
Installation complete.
$ cat spago.dhall
{-
Welcome to a Spago project!
You can edit this file as you like.
-}
{ name =
    "pscid"
, dependencies =
    [ "console", "prelude" ]
, packages =
    ./packages.dhall
, sources =
    [ "src/**/*.purs" ]
}
```

works when you have something in deps

**Comment by f-f** [comment]
Fix up in #282
